practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/service/PractitionerConfigurationProvider.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.ox.ndph.mts.practitioner_service.service;
+
+import java.nio.file.Files;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Component;
+
+import uk.ac.ox.ndph.mts.practitioner_service.configuration.PractitionerConfiguration;
+import uk.ac.ox.ndph.mts.practitioner_service.exception.ServerError;
+
+/**
+ * Provide Practitioner Configuration 
+ */
+@Component
+public class PractitionerConfigurationProvider {
+
+    private static final String ERROR_LOADING_CONFIGURATION = "Error while loading configuration file";
+
+    @Value("classpath:practitioner-configuration.json")
```

Comment:
should we be using the same approach as trial-config-service which gets the config file from git?

Reply:
@darrengsmithson, we would like to have all config fetch from a central store inside the trial scope. Spring cloud has the concept of a config store which is a good approach from micro-services system. 
But till that will be available, files are intermediate solution.